R/utils.R

```diff
@@ -1,23 +1,21 @@
+#' @import rlang
+#' @importFrom utils installed.packages
 
-`%||%` <- function(l, r) if (length(l) == 0) r else l
-
-is_string <- function(x) {
-  is.character(x) && length(x) == 1 && !is.na(x)
+`%|0|%` <- function(x, y) {
+  if (!length(x)) y else x
 }
 
-#' @importFrom utils installed.packages
-
```

Comment:
Why remove this?

Reply:
I'm not sure, maybe a typeo, I'll double check.